swagger/swagger-generator/generator-core/src/main/java/org/apache/servicecomb/swagger/generator/core/SwaggerGenerator.java

```diff
@@ -236,35 +231,13 @@ private boolean isInterfaceReactive(Class<?> interfaceCls) {
     return false;
   }
 
-  /**
-   * Whether this method should be processed as a swagger operation
-   * @return true if this isn't a swagger operation; otherwise, false.
-   */
-  protected boolean isSkipMethod(Method method) {
-    if (method.getDeclaringClass().getName().equals(Object.class.getName())) {
-      return true;
-    }
-    // skip static method
-    int modifiers = method.getModifiers();
-    if (Modifier.isStatic(modifiers)) {
-      return true;
-    }
-    // skip bridge method
-    if (method.isBridge()) {
-      return true;
-    }
-
-    ApiOperation apiOperation = method.getAnnotation(ApiOperation.class);
-    if (apiOperation != null && apiOperation.hidden()) {
-      return apiOperation.hidden();
-    }
-
-    return !context.canProcess(method);
+  protected boolean isSkipMethod(Method m) {
+    return !context.canProcess(m);
   }
 
   protected void scanMethods() {
     // 有时方法顺序不同，很不利于测试，所以先排序
-    List<Method> methods = Arrays.asList(cls.getMethods());
+    List<Method> methods = MethodUtils.findProducerMethods(cls);
     methods.sort(Comparator.comparing(Method::getName));
```

Comment:
Is the sorting operation here redundant? since the `MethodUtils#findProducerMethods` has sorted the methods

Reply:
yes, I am fixing it